src/App.css

```diff
@@ -185,17 +185,10 @@ html,
 }
 
 .material-icons.settings.folder {
```

Comment:
One thing, not really related with the PR.
Have you noticed that we are using two different folder icons? one on the settings and another on the InstallModal.
I think we should pick one, maybe the one that we are using on the InstallModal looks a bit better imo.
But we can do that in another task if you think this could increase the scope of this PR.

Reply:
yeah, I didn't know the reason so I just skipped that, I agree that a single icon in all palces would look better

I'm also thinking about adding another component `PathInputField` that would take care of the input with the folder icon, we use that in a lot of places, I think it's a good opportunity to simplify that

Reply:
I extracted a TextInputWithIconField component instead, maybe we can extract a PathInputField in the future, but not all path inputs behave the same so decided to go with a more generic input with icon